packages/esm-patient-search-app/src/patient-search-page/patient-banner/banner/patient-banner.component.tsx

```diff
@@ -82,6 +82,12 @@ const PatientBanner: React.FC<PatientBannerProps> = ({
 
   const isDeceased = !!patient.person.deathDate;
 
+  const fhirPatient = React.useMemo(() => {
+    return {
+      deceasedDateTime: patient.person.deathDate,
+    };
+  }, [patient]);
+
```

Comment:
only `deceasedDateTime` property is needed in both tags, so passing it alone is sufficient.